### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by sxnney** [sweetriverfish/LeagueAutoAcceptPy#1]
Couldn't find a way to message you. 

Can you add me on discord?

Sunney#7906

**Comment by sweetriverfish** [comment]
I don't really use Discord. What is it that you need?

**Comment by sweetriverfish** [comment]
Also, how did you even find this? lol

**Comment by sxnney** [comment]
I was making the same tool, but with PyAutoGUI and started searching for other's solutions and I felt dumb when I discovered League has a client API. 

Anyways I have project idea to make this even more cool.
Add feature to automatically select your champion, ban champions and etc.

We can work on it if you want.

**Comment by sxnney** [comment]
https://gist.github.com/Sunney-X/501f914681b598909903188ab0f34bae

This is what my version of it looks like.

**Comment by sweetriverfish** [comment]
I was thinking about extending it, but I'm not very interested.  

First: this going to be somewhat complicated when you start adding all the different champ select types and I'm a little too lazy lol  

Second: this wasn't the goal of this small script.  
The reason I created it is because me (and others) who have extreme high mmr and wait 20 minutes for blind pick queues pretty often sometimes want to do something during queue (for example pee, if it wasn't obvious yet) and we were either afraid to go as "queue might pop soon better stay for another few minutes" or we would go and then miss the queue (rip all those queues). A script that automatically accepts the queue buys just enough time for us to not worry about missing queue.  

And last: lol my Python skills are trash. You probably don't wanna work on it with me. To make it clear, this is the only Python script I ever made. I downloaded Python just for this and I never used it again for anything else.

Feel free to take the code and modify/extend it as you like, but I probably won't be helping much.
